src/components/Search/SearchResultsListItem.tsx

```diff
@@ -53,25 +68,51 @@ const styles = {
 function SearchResultsListItem({
   index,
   item,
-  searchType,
+  isEditable,
+  isBase,
+  inProgress,
+  setInProgress,
 }: SearchResultsListItemProps) {
+  const searchType = isBase ? 'base' : 'new';
+  const { handleToggle } = useCheckRevision(isBase, isEditable);
+
   const isChecked: boolean = useAppSelector((state) =>
```

Comment:
nit: name this `isCommittedChecked` so that the ternary below is clearer, and you can rename `isCheckedState` to `isChecked`.